com.vrcfury.vrcfury/Editor/VF/Service/ScaleFactorService.cs

```diff
@@ -25,7 +30,110 @@ public class ScaleFactorService {
         [VFAutowired] private readonly ForceStateInAnimatorService forceStateInAnimatorService;
         [VFAutowired] private readonly ClipBuilderService clipBuilder;
         
+        // Scaled Properties DirectTree
+        private int objectNumber = 0;
+        private BlendTree directTree = null;
+        private AnimationClip zeroClip = null;
+
+        // Scale Factor Fix
         private VFAFloat cached;
+
+        // Scaled Properties DirectTree
+        public void AddScaledPorperties(VFGameObject scaleReference, IEnumerable<(string ObjectPath, Type ComponentType, string PropertyName, object InitialValue)> properties) {

```

Comment:
This is great. Any chance you could move this to a new service file? This service is really just intended to be used for getting the ScaleFactor (which is more complex than it ought to be). This new logic is more like ScalePropertyCompensationService or something.

Reply:
Done